js/models/contact_model.js

```diff
@@ -34,6 +34,24 @@ angular.module('contactsApp')
 				return this.fullName() || this.org() || '';
 			},
 
+			firstName: function() {
+				var property = this.getProperty('n');
+				if (property) {
+					return property.value[1];
```

Comment:
From my pov we need to do this differently.
1. in case there are two contacts with the same first name we need to use the last name (or the additional first names???) as second sort criteria
2. in case no last name or last name is set we need to sort by displayname otherwise this will be chaotic
3. we should think about to adjust the name which is displayed: 
   - by last name: last name + first name + additional names
   - by first name: first name + additional names + last name

4. Open question: what shall we do with those contacts which have no N but only a FN - e.g. companies have that